52.md

```diff
@@ -26,11 +26,15 @@ The `.content` of these events should be a detailed description of the calendar
 
 The list of tags are as follows:
 * `d` (required) universally unique identifier (UUID). Generated by the client creating the calendar event.
-* `title` (required) title of the calendar event
+* `type` (required) type of the event, list of [_onsite_, _online_]
 * `start` (required) inclusive start date in ISO 8601 format (YYYY-MM-DD). Must be less than `end`, if it exists.
 * `end` (optional) exclusive end date in ISO 8601 format (YYYY-MM-DD). If omitted, the calendar event ends on the same date as `start`.
-* `location` (optional, repeated) location of the calendar event. e.g. address, GPS coordinates, meeting room name, link to video call
+* `address` (required if `type` is _onsite_) address of the calendar event following ISO 19160:2015 and splitting the elements in the array; the country is specified in ISO 3166
+* `place` (optional, permitted only if `type` is _onsite_) name of the place of the calendar event. e.g. univeristy, building, business, public space
 * `g` (optional) [geohash](https://en.wikipedia.org/wiki/Geohash) to associate calendar event with a searchable physical location
+* `url` (required if `type` is _online_) the url where the user can access the calendar event; it can be a nostr:naddr1xxx or a http(s)://xxx endpoint
```

Comment:
I see that in other NIPs custom tags are used when it is intended that they not be repeated; may it is better to stick wit the `r`.
Now thinking a about this I see the use of more resources, for example an online event could have a main presentation website and a direct access to the booking.
Maybe it could be a repeated tag like:

```
["r", "<url>", "description"]
```
Example:
```
["r", "https://myevent.com", "Webiste"]
["r", "https://myevent.com/book", "Sign up for the event"]
```